package/MDAnalysis/coordinates/PDB.py

```diff
@@ -465,6 +491,9 @@ class PDBWriter(base.WriterBase):
     The maximum frame number that can be stored in a PDB file is 9999 and it
     will wrap around (see :meth:`MODEL` for further details).
 
+    The CRYST1_ record specifies the unit cell. This record is set to
+    unitary values (cubic box with sides of 1 Å) if unit cell dimensions
+    are not set.
```

Comment:
As per my comment above, it might be good to be a bit more explicit to users here (i.e. "not set" in practical terms means either `None` or [0,0,0,0,0,0]).